embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/capture/crumbs/LegacyFragmentBreadcrumbDataSource.kt

```diff
@@ -0,0 +1,90 @@
+package io.embrace.android.embracesdk.capture.crumbs
+
+import io.embrace.android.embracesdk.arch.DataCaptureService
+import io.embrace.android.embracesdk.config.ConfigService
+import io.embrace.android.embracesdk.internal.clock.Clock
+import io.embrace.android.embracesdk.logging.InternalEmbraceLogger
+import io.embrace.android.embracesdk.logging.InternalStaticEmbraceLogger
+import io.embrace.android.embracesdk.payload.FragmentBreadcrumb
+import io.embrace.android.embracesdk.utils.filter
+import java.util.Collections
+
+/**
+ * Captures fragment breadcrumbs.
+ */
+internal class LegacyFragmentBreadcrumbDataSource(
```

Comment:
The functionality of the legacy services are validated by... the integration tests?

Reply:
If there's existing test coverage yes. I think this one is covered by existing unit test coverage.